python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -122,7 +132,7 @@ def validate(self):
         assert self.modelPsfMatch.kernel.active.alardNGauss == 1
 
     @classmethod
-    def getGaapResultName(cls, sF: float, sigma: float, name: Optional[str] = None) -> str:
+    def getGaapResultName(cls, sF: float, sigma: Union[float, str], name: Optional[str] = None) -> str:
```

Comment:
Is it really necessary to shorten the scale factor variable name to `sF` rather than `scaleFactor`? I think it hinders readability, even if it's technically style guide compliant.

Reply:
In the updated branch, `sF` is used only in this private method, or as temporary variables in comprehensions. The next ticket that will be merged after this one will have some significant refactoring, and I'll include a better variable name in there.